diff --git go/private/sdk.bzl go/private/sdk.bzl
--- go/private/sdk.bzl
+++ go/private/sdk.bzl
@@ -69,7 +69,8 @@ def _go_download_sdk_impl(ctx):
             ctx.report_progress("Finding Go SHA-256 sums")
         ctx.download(
             url = [
+                "https://golang.org/dl/versions.json",
                 "https://golang.org/dl/?mode=json&include=all",
                 "https://golang.google.cn/dl/?mode=json&include=all",
             ],
             output = "versions.json",